tensorflow_serving/model_servers/http_rest_api_util.cc

```diff
@@ -33,6 +33,12 @@ void AddHeaders(std::vector<std::pair<string, string>>* headers) {
   headers->push_back({"Content-Type", "application/json"});
 }
 
+void AddCORSHeaders(std::vector<std::pair<string, string>>* headers) {
+  headers->push_back({"Access-Control-Allow-Origin", "*"});
+  headers->push_back({"Access-Control-Allow-Methods", "POST, GET, OPTIONS"});
```

Comment:
Remove OPTIONS now it's directly handled or you need allow OPTIONS when Origin is missing.